README.md

```diff
@@ -1,21 +1,25 @@
 ## BASH Script Storage Project
 
-This is a qucik BASH scripting project that determines how much storage you have on your machine. This is one of the first projects I have worked on while on my jounrey to learn Linux and BASH Scripting.  
+This is a BASH scripting project that determines how much storage you have on your machine. 
+
+## Requirements
```

Comment:
Create a requirements section. This makes it easier for the consumer to determine if they can invoke the script and/or make changes.